Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class NetqRabbitRpcClient : IRpcClient
+    {
+        private readonly INetqRabbitRpcConnection<Func<Type, object>> _netqRpcRabbitConnection;
+        private readonly ILogger _logger;
+
+        public NetqRabbitRpcClient(INetqRabbitRpcConnection<Func<Type, object>> netqRpcRabbitConnection, ILogger logger)
+        {
+            _logger = logger;
+            _netqRpcRabbitConnection = netqRpcRabbitConnection;
+        }
+
+        public TRespond Request<TRequest, TRespond>(TRequest request)
+        where TRequest : class
+        where TRespond : class
+        {
+            if (_netqRpcRabbitConnection.RpcMessageType == RpcMessageType.Abstract)
+            {
+                _logger.LogDebug($"Requesting an abstract type request of type: {typeof(TRequest)} and expecting to receive a respond of type: {typeof(TRespond)}");
+                return _netqRpcRabbitConnection.Bus.Request<TRequest, TRespond>(request);
+            }
+            else if (_netqRpcRabbitConnection.RpcMessageType == RpcMessageType.Concrete)
```

Comment:
`Concrete` strategy should be the default one, I think that it's better to put an `else` here.

Reply:
Im already throwing an exception so there is not default one.
I dont want to add default one it seem to important to me and I want the user to know that the RpcMessageType he gave me is not going to be used